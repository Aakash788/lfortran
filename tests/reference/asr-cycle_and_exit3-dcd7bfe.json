{
    "basename": "asr-cycle_and_exit3-dcd7bfe",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/cycle_and_exit3.f90",
    "infile_hash": "05289c231a04f0752effeea6ff1251767ccd54cd98603687066e107b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-cycle_and_exit3-dcd7bfe.stdout",
    "stdout_hash": "f4860027228f7a7fe6cd9025faa5ebfcf9b12956babe2af2d346d0b5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}