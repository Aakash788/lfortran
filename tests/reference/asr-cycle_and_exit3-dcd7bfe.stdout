(TranslationUnit
    (SymbolTable
        1
        {
            loop_and_block:
                (Program
                    (SymbolTable
                        2
                        {
                            i:
                                (Variable
                                    2
                                    i
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    loop_and_block
                    []
                    [(DoLoop
                        l1
                        ((Var 2 i)
                        (IntegerConstant 1 (Integer 4 []))
                        (IntegerConstant 10 (Integer 4 []))
                        ())
                        [(Print
                            ()
                            [(StringConstant
                                "l1"
                                (Character 1 2 () [])
                            )
                            (Var 2 i)]
                            ()
                            ()
                        )
                        (BlockCall
                            -1
                            2 b1
                        )
                        (If
                            (IntegerCompare
                                (Var 2 i)
                                Gt
                                (IntegerConstant 2 (Integer 4 []))
                                (Logical 4 [])
                                ()
                            )
                            [(Print
                                ()
                                [(StringConstant
                                    "error l1"
                                    (Character 1 8 () [])
                                )]
                                ()
                                ()
                            )]
                            []
                        )]
                    )
                    (If
                        (IntegerCompare
                            (Var 2 i)
                            Eq
                            (IntegerConstant 3 (Integer 4 []))
                            (Logical 4 [])
                            ()
                        )
                        [(Print
                            ()
                            [(StringConstant
                                "pass"
                                (Character 1 4 () [])
                            )]
                            ()
                            ()
                        )]
                        []
                    )]
                )
        })
    []
)
